src/browser/pages/vscode.html

```diff
@@ -39,8 +39,43 @@
   <body aria-label=""></body>
 
   <!-- Startup (do not modify order of script tags!) -->
-  <script data-cfasync="false" src="{{CS_STATIC_BASE}}/dist/pages/vscode.js"></script>
-  <script data-cfasync="false" src="{{CS_STATIC_BASE}}/dist/register.js"></script>
+  <script>
```

Comment:
Wait this isn't supposed to be in here? I thought we could keep it in `vscode.ts`?

Reply:
Yup, we can remove all this.